dotnet-authserver/src/TeacherIdentity.AuthServer/Services/DqtApi/TeacherNameChangeRequest.cs

```diff
@@ -0,0 +1,11 @@
+namespace TeacherIdentity.AuthServer.Services.DqtApi;
+
+public record TeacherNameChangeRequest
```

Comment:
No way to make sure these fields are non-null when using object instantiator, but not a big deal I guess